corehq/apps/hqadmin/management/commands/preindex_everything.py

```diff
@@ -56,7 +56,11 @@ def handle(self, num_pool, username, **options):
         head = git_snapshot['commits'][0]
 
         if options['check']:
-            exit(0 if get_preindex_complete(head) else 1)
+            if get_preindex_complete(head):
+                print("Preindex is complete")
+                exit(0)
+            print("Preindex is not yet complete")
```

Comment:
In rare cases like yesterday it can also mean the preindex is not only not yet complete, but stopped even running. This change is already an improvement, but while we're here I might add

```suggestion
            print("Preindex is not yet complete.")
            print("")
            print("  It could either still be running (most common),")
            print("  or it could have died without completing (less common).")
```

Reply:
thanks, committed!